mmf/common/registry.py

```diff
@@ -21,6 +21,7 @@
 - Register a optimizer: ``@registry.register_optimizer``
 - Register a scheduler: ``@registry.register_scheduler``
 - Register a decoder: ``@registry.register_decoder``
+- Register a transformer backend: ``@registry.register_transformer_backend``
```

Comment:
Suggestion: I think it would be better to not use registry for this as it might need to be used independent of registry. I would suggest that we follow the route of ImageEncoder style factory pattern and provide a `build_transformer_backend` function which can then return the proper backend module. This is something that will probably stay in core for quite some time and thus should be easier handled via that API as it won't require calling `setup_imports` and other things required to initialize MMF. We should think of this as one of the module that MMF provides.

Reply:
I made the change to ImageEncoder style. However, if having these in registry is not an issue it's better to have them in registry. Let me know your thoughts and I revert that change.